spec/lib/guard/notifier_spec.rb

```diff
@@ -41,4 +41,42 @@
       end
     end
   end
+
+  describe "backward compatibility" do
```

Comment:
Not necessary to describe as "backward compatibility" (I wasn't clear in what I said) - it's expected for the `notify()` method to work with single+multiple params (for as long as the Notifier class will exist in Guard).